MediaBrowser.Controller/Entities/TV/Series.cs

```diff
@@ -318,20 +318,13 @@ public async Task RefreshAllMetadata(MetadataRefreshOptions refreshOptions, IPro
 
                 cancellationToken.ThrowIfCancellationRequested();
 
-                var skipItem = false;
-
-                var episode = item as Episode;
-
-                if (episode != null
-                    && refreshOptions.MetadataRefreshMode != MetadataRefreshMode.FullRefresh
-                    && !refreshOptions.ReplaceAllMetadata
-                    && episode.IsMissingEpisode
-                    && episode.LocationType == LocationType.Virtual
-                    && episode.PremiereDate.HasValue
-                    && (DateTime.UtcNow - episode.PremiereDate.Value).TotalDays > 30)
-                {
-                    skipItem = true;
-                }
+                bool skipItem = item is Episode episode
+                                && refreshOptions.MetadataRefreshMode != MetadataRefreshMode.FullRefresh
+                                && !refreshOptions.ReplaceAllMetadata
+                                && episode.IsMissingEpisode
+                                && episode.LocationType == LocationType.Virtual
+                                && episode.PremiereDate.HasValue
+                                && (DateTime.UtcNow - episode.PremiereDate.Value).TotalDays > 30;
```

Comment:
this should be an if statement imo

Reply:
I disagree, since the only thing the if statement would do is assign the variable anyways

Reply:
The logical statement looks almost identical to the if statement. And I personally find this version cleaner and easier to read.